src/vmm/src/default_syscalls/filters.rs

```diff
@@ -109,6 +109,10 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
             allow_syscall(libc::SYS_timerfd_settime),
             allow_syscall(libc::SYS_write),
             allow_syscall(libc::SYS_writev),
+            #[cfg(target_env = "gnu")]
+            allow_syscall(libc::SYS_getpid),
```

Comment:
Can you rearrange these so that the list remains in alphabetical order? Makes it easier to find a syscall when scrolling through it.

Reply:
Sure. I didn't notice the ordering

Reply:
done